### DISCUSSION TITLE
API or no API

### DISCUSSION THREAD

**Discussion by Tonlaar** [OlivierZal/com.melcloud#276] [Q&A]
I'm more than happy with this integration!
Primarily use it (in combination with the CV room thermostat) to control the airco or heat pump threshold.  
I try to understand how invasive this "control" could be for the Mitsubishi Electric installation. Therefore just a question. 
Is this integration based on an API or do you use some other method to communicate with the MELCloud app?
Kind  regards,
Ton

**Comment by OlivierZal** [comment]
Hi @Tonlaar,

Thanks for your feedback!

I'm not sure I understand "control invasion", but to answer your question: the app relies only on the official MELCloud API.

**Comment by Tonlaar** [comment]
with "invasive” I mean, can I damage the outside unit by manipulating the MELCloudapp. 
Do you have some more information on the official MELCloud API. I try to understand under what circumstances is it allowed to use the API
Kind regards
Ton

**Reply by OlivierZal** [reply]
The risk is the same than using the official MELCloud app, so if you don’t send strange, intempestive commands it shouldn’t hurt. 

There’s no official documentation of this API, everything has been deduce from queries sent by the MELCloud website.
